.azure-pipelines/client.yml

```diff
@@ -13,112 +13,44 @@ jobs:
       vmImage: 'vs2017-win2016'
 
     steps:
-      - script: |
-         "%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\sn.exe" -Vr *,31bf3856ad364e35
-        displayName: 'Allow unsigned build tool'
-
-      - task: MSBuild@1
-        displayName: 'Init Build'
-        inputs:
-          solution: build.proj
-          msbuildArguments: '/t:Init /flp:LogFile=msbuild.Initlog;Verbosity=normal /p:UseLatest=$(build.useLatest)'
-
-      - task: MSBuild@1
-        displayName: 'Build NuGet Package'
-        inputs:
-          solution: build.proj
-          msbuildArguments: '/t:CreateNugetPackage /p:Scope=$(scope) /p:Configuration=Release /clp:ShowTimestamp /flp:Summary;Verbosity=minimal;LogFile=FullBuild.sum /flp1:warningsonly;logfile=FullBuild.wrn /flp2:errorsonly;logfile=FullBuild.err /p:DebugTrace=$(system.debug) /p:CodeSign=false /p:AuthenticodeSign=false /p:RunGlobalReadinessTools=false /p:EnablePreBuildTarget=false /p:EnablePostBuildTarget=false /p:SkipTests=true'
-
-      - script: |
-         "%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\sn.exe" -Vu *,31bf3856ad364e35
-        displayName: 'Clean up unsigned build tool allowance'
-        condition: always()
+      - script: dotnet pack src/SDKs/data-plane.proj /p:PackageOutputPath=$(Build.ArtifactStagingDirectory)
+        displayName: 'Build & Package'
 
       - task: PublishBuildArtifacts@1
         displayName: 'Publish Artifacts'
-        inputs:
-          PathtoPublish: 'binaries\packages'
-          ArtifactName: packages
 
   - job: 'Test_Windows'
 
     pool:
       vmImage: 'vs2017-win2016'
 
     steps:
-      - script: |
-         "%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.1 Tools\sn.exe" -Vr *,31bf3856ad364e35
-        displayName: 'Allow unsigned build tool'
-
-      - task: MSBuild@1
-        displayName: 'Init Build'
-        inputs:
-          solution: build.proj
-          msbuildArguments: '/t:Init /flp:LogFile=msbuild.Initlog;Verbosity=normal /p:UseLatest=$(build.useLatest)'
+      - script: dotnet test src/SDKs/data-plane-tests.proj --logger trx
```

Comment:
We should parameterize scope here probably.

Reply:
This will likely need to be a follow-up change so we can correctly setup the build pipeline.